developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -47,7 +52,8 @@ export interface CompilerResultExtra {
   targets: number;
   kvksFilename?: string;
   displayMapFilename?: string;
-  stores?: CompilerResultExtraStore[];
+  stores: CompilerResultExtraStore[];
```

Comment:
we never need this to be optional -- even if empty, it should be an array not null.